admin/sales_analytics/metrics.py

```diff
@@ -0,0 +1,113 @@
+from admin.base.settings import ENTRY_POINTS
+from admin.sales_analytics.utils import get_entry_point, get_sorted_index
+from datetime import datetime, timedelta
+from framework.mongo import database as db
+
+
+def get_user_count(db=db, entry_points=ENTRY_POINTS):
```

Comment:
The name of this file should conform to the module standards (mostly Django standards) This looks like either a `utils.py` or a `serializers.py` file.